source/options.js

```diff
@@ -75,6 +75,7 @@ export const buildOptions = (helpText, options) => {
 		inferType: false,
 		input: 'string',
 		help: helpText,
+		version: foundPackage?.packageJson.version || 'No version found',
```

Comment:
Using `||` over `??` because in #256, the normalization turns `pkg.version` into an empty string.